dhcpv4/ztpv4/ztp.go

```diff
@@ -15,39 +17,60 @@ type VendorData struct {
 
 var errVendorOptionMalformed = errors.New("malformed vendor option")
 
-// ParseVendorData will try to parse dhcp4 options looking for more
-// specific vendor data (like model, serial number, etc).
-func ParseVendorData(packet *dhcpv4.DHCPv4) (*VendorData, error) {
+func parseV4VIVC(vd *VendorData, packet *dhcpv4.DHCPv4) error {
+	vivc := packet.VIVC()
+	for _, id := range vivc {
+		if id.EntID == iana.EntIDCiscoSystems {
+			vd.VendorName = "Cisco Systems"
```

Comment:
Can the "Cisco Systems" string come from the IANA package?

Reference comment below.